current/standard-derivatives.md

```diff
@@ -28,12 +28,11 @@ import io.archivesunleashed.matchbox._
 sc.setLogLevel("INFO")
```

Comment:
line 15 - should we add the `code` command (inline) for concatenation, as graph pass command is written directly below the paragraph?